hack/dump_cache.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script sends a USR2 signal to a running kueue binary in the kueue-system namespace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+NAMESPACE=${NAMESPACE:-kueue-system}
+DEBUG_IMAGE=${DEBUG_IMAGE:-gcr.io/k8s-staging-kueue/debug:main}
+
+leader=$(kubectl get lease -n ${NAMESPACE} -o jsonpath='{.items[0].spec.holderIdentity}' | cut -d '_' -f 1)
```

Comment:
Should we obtain the leader name from `c1f6bfd2.kueue.x-k8s.io`? If we deploy the manager for custom jobs separate from the kueue-manager, this script probably doesn't work well.

https://github.com/kubernetes-sigs/kueue/blob/7bf4e920808eaac7324bee05c636721861a0280c/config/components/manager/controller_manager_config.yaml#L12

Reply:
Done